NextcloudTalk/NCChatViewController.m

```diff
@@ -3348,6 +3348,23 @@ - (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSIntege
     if ([tableView isEqual:self.autoCompletionView]) {
         return 0;
     }
+
+    NSDate *date = [_dateSections objectAtIndex:section];
+    NSMutableArray *messages = [_messages objectForKey:date];
+
+    NSUInteger idx = [messages indexOfObjectPassingTest:^BOOL(NCChatMessage *message, NSUInteger idx, BOOL * _Nonnull stop) {
+        if (![message isUpdateMessage]) {
+            *stop = YES;
+            return YES;
+        }
+
+        return NO;
+    }];
```

Comment:
Nice idea to hide empty sections ;)
Maybe we could reuse the function `newMessagesContainVisibleMessages()` and rename it to something like `messagesContainVisibleMessages()`, what do you think? :)

Reply:
Oh you’re right, good idea  👍 I’ll update the PR